src/components/common/Layout/Layout.js

```diff
@@ -17,6 +18,19 @@ const Layout = ({ children }) => (
         <SEO />
         <GlobalStyles />
         {children}
+        <CookieConsent
+          location="bottom"
+          buttonText="Accept"
+          cookieName="gatsby-gdpr-google-analytics"
+          buttonStyle={{ background: '#fafafa', fontSize: '13px' }}
+          sameSite="lax"
+        >
+          We use cookies and other tracking technologies to improve your
```

Comment:
@hasanagh, I believe we already have EN and FR translation activated. Please check with @hagoptaminian to make this text available in both languages. Thanks!

Reply:
Correct - you can just `import { useTranslation }` from `react-i18next`, use `const { t } = useTranslation()` at the top of this component, and then wrap both `buttonText` and the cookie text with `{t('')}`. Make sure to update the translation pairs in `/src/config/i18n`.